table/queries/merged_reduced_scans.sql

```diff
@@ -90,13 +90,22 @@ CREATE TEMP FUNCTION ClassifyError(error STRING, source STRING, template_match B
 
     # Content verification failures
     WHEN (source = "ECHO" AND ENDS_WITH(error, "EOF")) THEN "content/response_mismatch" # Echo
+    # Hyperquack v1 errors
     WHEN (error = "Incorrect echo response") THEN "content/response_mismatch" # Echo
     WHEN (error = "Received response") THEN "content/response_mismatch" # Discard
     WHEN (error = "Incorrect web response: status lines don't match") THEN "content/status_mismatch" # HTTP/S
     WHEN (error = "Incorrect web response: bodies don't match") THEN "content/body_mismatch" # HTTP/S
     WHEN (error = "Incorrect web response: certificates don't match") THEN "content/tls_mismatch" # HTTPS
     WHEN (error = "Incorrect web response: cipher suites don't match") THEN "content/tls_mismatch" # HTTPS
     WHEN (error = "Incorrect web response: TLS versions don't match") THEN "content/tls_mismatch" # HTTPS
+    # Hyperquack v2 errors
+    WHEN (error = "echo response does not match echo request") THEN "content/response_mismatch" # Echo
+    WHEN (error = "discard response is not empty") THEN "content/response_mismatch" # Discard
+    WHEN (error = "Status lines does not match") THEN "content/status_mismatch" # HTTP/S
```

Comment:
One thing we may want to make clear either in the documentation or somewhere in the pipeline is that Hyperquackv2 (and v1) only outputs the first mismatch it finds. Cases of "content/status_mismatch" frequently also have a body mismatch. This also means that the TLS mismatch cases are especially interesting as they possibly show cases of transparent TLS interception for certain domains.

Reply:
Good point, probably the best place to put that would be in the docs. I'll have to think about how best to explain.